lib/ansible/utils/hashing.py

```diff
@@ -93,3 +93,6 @@ def md5(filename):
         raise ValueError('MD5 not available.  Possibly running in FIPS mode')
     return secure_hash(filename, _md5)
 
+from binascii import crc32 as crc32
+def crc32(data):
```

Comment:
Looking at hashing.py, I take back the comment about the naming convention, since there doesn't really seem to be one there. So 'crc32_s' is fine.